nuget/lib/BlingFireUtils.cs

```diff
@@ -1,26 +1,40 @@
+// <copyright file="BlingFireUtils.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+//     Licensed under the MIT License.
+// </copyright>
+
 using System;
 using System.Collections.Generic;
-using System.Text;
 using System.Runtime.InteropServices;
+using System.Text;
 
 /// <summary>
 /// This is C# interface for blingfiretokdll.dll
 /// </summary>
 namespace BlingFire
 {
-    public static class BlingFireUtils
+    public static class BlingFire

```

Comment:
We cannot change this name. It was already published.

Please ensure no public interface change.

Reply:
My bad, will fix it.